sensitive-banner-static/res/common-banner.js

```diff
@@ -15,7 +17,9 @@ $( function () {
 		if ( Math.random() < 0.01 ) {
 			$( '#WMDE_Banner-close-ct' ).attr( 'src', replaceWikiVars( 'https://spenden.wikimedia.de/piwik/piwik.php?idsite=1&url=https://spenden.wikimedia.de/banner-closed/{{{BannerName}}}&rec=1' ) );
 		}
-		mw.centralNotice.hideBanner();
+		if ( mw && mw.centralNotice ) {
```

Comment:
this does not seem to be enough. Maybe doing it by checking if mw is defined, as I did in `onMediaWiki` function in https://github.com/wmde/FundraisingBanners/commit/c572f14632c623e8ab03abb05dd2cf2d96ad1fb0 ?